app.js

```diff
@@ -136,17 +135,16 @@ if (Config.crashguard) {
 
 global.Sockets = require('./sockets');
 
-exports.listen = function (port, bindAddress, workerCount) {
-	Sockets.listen(port, bindAddress, workerCount);
+exports.listen = function (port) {
```

Comment:
?

the `if (require.main === module) {` block should be the only block reading `process.argv`.